src/main/resources/db.changelog/changes/equipment.yaml

```diff
@@ -40,7 +40,8 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: eq_created
-                  type: CURRENT_TIMESTAMP
```

Comment:
We can't modify existing changesets, please create a new ones for this purpose

Reply:
this is not my table I didn't do this

Reply:
Look at the master branch yesterday I added new changesets. you can take a look. You can write a new changeSet without creating a new file.

Reply:
I inserted two column into employee table using changeset like you said bro